lib/repositories/rocket.py

```diff
@@ -50,7 +52,9 @@ async def create_rocket(self, rocket_option: str = "Calisto") -> "InsertOneResul
         else:
             return InsertOneResult(acknowledged=True, inserted_id=None)
 
-    async def update_rocket(self, rocket_option: str = "Calisto") -> "Union[int, None]":
+    async def update_rocket(
+        self, rocket_option: str = "Calisto"
```

Comment:
Self note: Need to discuss default values for all variables that get fed into the API. Mabey implement some random name generator or just keep the field an empty string "". No comments necessary now.

Reply:
Agreed!

Me and @Gui-FernandesBR had a discussion about default values in the past and decided to keep them to a minimum (to avoid whenever possible), but maybe for the integration between API and web-client we might have to discuss it again.